lang/python/python3-importlib-metadata/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=importlib_metadata
+PKG_VERSION:=0.17
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/i/importlib_metadata
+PKG_HASH:=a9f185022cfa69e9ca5f7eabfd5a58b689894cb78a11e3c8c89398a8ccbb8e7f
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-importlib-metadata
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Like importlib_resources only for metadata.
+  URL:=https://gitlab.com/python-devs/importlib_metadata
```

Comment:
@jefferyto What do you think that there should be Git repository instead of [documentation](https://importlib-metadata.readthedocs.io/en/latest/)? The URL for Documentation is listed actually as homepage on pypi.